middle_end/flambda/closure_conversion.ml

```diff
@@ -170,7 +170,7 @@ let lambda_const_int i : Lambda.structured_constant =
 
 let rec close t env (lam : Lambda.lambda) : Flambda.t =
   match lam with
-  | Lvar id ->
+  | Lvar id | Lmutvar id ->
```

Comment:
Here you should be able to simplify the code below: the `Lvar` case should use `find_var`, and the `Lmutvar` case should `find_mutable_var` directly.